bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -122,6 +122,8 @@ private func dummy() {
     _ = NSLocalizedString("Double tap to select a video", comment: "")
     _ = NSLocalizedString("Double tap to select an audio file", comment: "")
     _ = NSLocalizedString("Double tap to select an image", comment: "")
+    _ = NSLocalizedString("Double tap to select default font size", comment: "")
+    _ = NSLocalizedString("Double tap to select font size", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.native.js#L147:L147)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.